zmon-actuator/src/main/java/org/zalando/zmon/actuator/metrics/MetricsWrapper.java

```diff
@@ -17,43 +17,43 @@
 
 public class MetricsWrapper {
 
-    private static final Logger LOGGER = getLogger(MetricsWrapper.class);
+  private static final Logger LOGGER = getLogger(MetricsWrapper.class);
     private static final String UNKNOWN_PATH_SUFFIX = "/unmapped";
 
-    private final MetricRegistry metricRegistry;
+    private final MeterRegistry meterRegistry;
 
     @Autowired
-    public MetricsWrapper(final MetricRegistry metricRegistry) {
-        this.metricRegistry = metricRegistry;
+    public MetricsWrapper(final MeterRegistry meterRegistry) {
+        this.meterRegistry = meterRegistry;
     }
 
     public void recordClientRequestMetrics(final HttpServletRequest request, final String path, final int status,
                                            final long time) {
-        String suffix = getFinalStatus(request);
+      final String suffix = this.getFinalStatus(request);
 
-        String metricName = metricNameFrom("zmon.response", status, request.getMethod(), suffix);
-        submitToTimer(metricName, time);
+      final String metricName = metricNameFrom("zmon.response", status, request.getMethod(), suffix);
```

Comment:
Shouldn't this use tags now instead of constructing a dynamic name?

Reply:
True